CHANGELOG.md

```diff
@@ -53,6 +55,14 @@ Changelog
   | moved class | `CellIterator.OSMEntityFilter` | `oshdb-util/oshdb.osm.OSMEntityFilter` |
   | moved class | `oshdb-api.generic.function` | `oshdb-util/oshdb.util.function` |
   | renamed enum constants | `ComputeMode.LocalPeek`, `….ScanQuery`, `….AffinityCall` | `ComputeMode.LOCAL_PEEK`, `….SCAN_QUERY`, `….AFFINITY_CALL` |
+* if you are using the ignite backend, you need to add the following dependency to your project:
+  ```xml
+    <dependency>
+      <groupId>org.heigit.ohsome</groupId>
+      <artifactId>oshdb-backend-ignite</artifactId>
```

Comment:
:thinking: shouldn't it be `oshdb-api-ignite`? Or do you mean if you're running an ignite backend and don't want to just connect to an ignite instance? Maybe you should mention `oshdb-api-ignite` as well.

Reply:
oops, yes. At first I called the module `-backend-` but figured that `-api-` is a better name.